src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -1754,15 +1785,15 @@ public void uploadOutputs(
       return;
     }
 
-    if (remoteOptions.remoteCacheAsync
+    if (remoteOptions.remoteCacheAsync != RemoteCacheAsync.FALSE
         && !action.getSpawn().getResourceOwner().mayModifySpawnOutputsAfterExecution()) {
       backgroundTaskExecutor.execute(
           () -> {
             try {
               doUploadOutputs(action, spawnResult, onUploadComplete);
             } catch (ExecException e) {
               reportUploadError(e);
-            } catch (InterruptedException ignored) {
+            } catch (InterruptedException e) {
```

Comment:
Please change this back to silence an IDE warning

Reply:
Done.